docs/UserGuide.md

```diff
@@ -175,7 +175,19 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.<br>
+
+**Q**: How to install Java 17<br>
+**A**: Download the Java 17 [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html). Then follow the [installation guide](https://docs.oracle.com/en/java/javase/17/install/overview-jdk-installation.html).<br>
+
+**Q**: How to know if you have Java 17<br>
+**A**: Open your command terminal and enter `java --version`. The first line should display `java` followed by the `version number`.<br>
+
+**Q**: Is there a limit to the number of patients/appointments we can add?<br>
+**A**: As of right now, We do not have a limit to the number of patients/appointments but is dependent on the hardware specification.<br>
+
+**Q**: Do you support non-English inputs?<br>
+**A**: We do not support non-English inputs, but we are working on it!<br>
```

Comment:
what if you did 
```
## FAQ

### **Q**: How do I transfer my data to another Computer?
- **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
- **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.

### **Q**: How to install Java 17
- **A**: Download the Java 17 [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html). Then follow the [installation guide](https://docs.oracle.com/en/java/javase/17/install/overview-jdk-installation.html).

### **Q**: How to know if you have Java 17
- **A**: Open your command terminal and enter `java --version`. The first line should display `java` followed by the `version number`.

### **Q**: Is there a limit to the number of patients/appointments we can add?
- **A**: As of right now, We do not have a limit to the number of patients/appointments but is dependent on the hardware specification.

### **Q**: Do you support non-English inputs?
- **A**: We do not support non-English inputs, but we are working on it!

```
It will show up in the `toc` under `FAQs` so the users can quickly find the FAQ instead of not scrolling and then lodging a ticket because they didnt rtfm

![image](https://github.com/user-attachments/assets/32eaa873-e8eb-4846-a175-c495da6b0328)


You'll get something like this:

## FAQ

### **Q**: How do I transfer my data to another Computer?
- **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
- **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.

### **Q**: How to install Java 17
- **A**: Download the Java 17 [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html). Then follow the [installation guide](https://docs.oracle.com/en/java/javase/17/install/overview-jdk-installation.html).

### **Q**: How to know if you have Java 17
- **A**: Open your command terminal and enter `java --version`. The first line should display `java` followed by the `version number`.

### **Q**: Is there a limit to the number of patients/appointments we can add?
- **A**: As of right now, We do not have a limit to the number of patients/appointments but is dependent on the hardware specification.

### **Q**: Do you support non-English inputs?
- **A**: We do not support non-English inputs, but we are working on it!

![image](https://github.com/user-attachments/assets/7b967d3d-46b8-4d56-a01e-389151eab1a9)

Reply:
or if the `Q: ...` is too big, make the header more sub (add more `#`)

![image](https://github.com/user-attachments/assets/49ba70ea-24e9-4859-b76e-4aa26bee64e9)